src/kiwixapp.cpp

```diff
@@ -40,17 +40,14 @@ KiwixApp::KiwixApp(int& argc, char *argv[])
         QMessageBox::critical(nullptr, "Translation error", e.what());
         return;
     }
+
 #if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
-    m_qtTranslator.load(QLocale(), "qt", "_",
-                        QLibraryInfo::location(QLibraryInfo::TranslationsPath));
+    QString path = QLibraryInfo::location(QLibraryInfo::TranslationsPath);
 #else
-    m_qtTranslator.load(QLocale(), "qt", "_",
-                        QLibraryInfo::path(QLibraryInfo::TranslationsPath));
+    QString path = QLibraryInfo::path(QLibraryInfo::TranslationsPath);
 #endif
-    installTranslator(&m_qtTranslator);
-
-    m_appTranslator.load(QLocale(), "kiwix-desktop", "_", ":/i18n/");
-    installTranslator(&m_appTranslator);
+    (void) loadAndInstallTranslations(m_qtTranslator, "qt", path);
+    (void) loadAndInstallTranslations(m_appTranslator, "kiwix-desktop", ":/i18n/");
```

Comment:
Since it is not a matter of defining an API that will be hard to change once we release it, I think that at this point it is cleaner to change the return type of `loadAndInstallTranslations()` to `void`.